src/zenml/cli/stack_recipes.py

```diff
@@ -0,0 +1,1013 @@
+
+    `zenml stack_recipe deploy stack_recipe_name` has to be called with the same relative
+    path before the destroy command.
+
+    Args:
+        ctx: The click context.
+        git_stack_recipes_handler: The GitStackRecipesHandler instance.
+        stack_recipe_name: The name of the stack_recipe.
+        path: The path at which you want to install the stack_recipe(s).
+        log_level: Choose one of TRACE, DEBUG, INFO, WARN or ERROR (case insensitive)
+            as log level for the deploy operation.
+
+    Raises:
+        ModuleNotFoundError: If the recipe is found at the given path.
+    """
+    cli_utils.warning(ALPHA_MESSAGE)
+
+    stack_recipes_dir = Path(os.getcwd()) / path
+
+    if sys.platform == "win32":
+        logger.info(
+            "If you are running stack_recipes on Windows, make sure that you have "
+            "an associated application with executing .sh files. If you don't "
+            "have any and you see a pop-up during 'zenml stack_recipe run', we "
+            "suggest to use the Git BASH: https://gitforwindows.org/"
+        )
+
+    try:
+        _ = git_stack_recipes_handler.get_stack_recipes(stack_recipe_name)[0]
+    except KeyError as e:
+        cli_utils.error(str(e))
+    else:
+        stack_recipe_dir = stack_recipes_dir / stack_recipe_name
+        local_stack_recipe = LocalStackRecipe(
+            stack_recipe_dir, stack_recipe_name
+        )
+
+        if not local_stack_recipe.is_present():
+            raise ModuleNotFoundError(
+                f"The recipe {stack_recipe_name} "
+                "has not been pulled at the specified path. "
+                f"Run `zenml stack recipe pull {stack_recipe_name}` "
+                f"followed by `zenml stack recipe deploy {stack_recipe_name}` first."
+            )
+
+        try:
+            # check if terraform is installed
+            tf_client = Terraform(str(local_stack_recipe.path))
+            try:
+                tf_client.check_installation()
+            except RuntimeError as e:
+                cli_utils.error(str(e))
+
+            # set terraform log level
+            tf_client.set_log_level(log_level=log_level)
+
+            # Telemetry
+            track_event(
+                AnalyticsEvent.DESTROY_STACK_RECIPE,
+                {"stack_recipe_name": stack_recipe_name},
+            )
+            # use the terraform client to call destroy on the recipe
+            tf_client.destroy()
+
+            cli_utils.declare(
+                "\n" + "Your active stack might now be invalid. Please run:"
+            )
+            text = Text("zenml stack describe", style="markdown.code_block")
+            cli_utils.declare(text)
+            cli_utils.declare(
+                "\n" + "to investigate and switch to a new stack if needed."
+            )
+
+        except python_terraform.TerraformCommandError as e:
+            force_message = ""
+            if stack_recipe_name == "aws_minimal":
```

Comment:
This is a bit too hard-coded IMO. We shouldnt have any recipe specific logic in here. We can for now keep it in but remove it later

Reply:
I completely agree but the eks terraform module currently doesn't get cleaned up properly. I first thought of adding a `--force`  option with logic that can destroy the eks module resources and then clean up the rest of the recipe. But owing to the very risky nature of the commands I decided against adding it to the CLI in any form and instead, went ahead with this very specific warning. 

I'll take another shot at resolving the `eks` problem and if resolved, this error message can be removed.